plot_app/configured_plots.py

```diff
@@ -443,6 +443,7 @@ def generate_plots(ulog, px4_ulog, db_data, vehicle_data, link_to_3d_page,
     data_plot.add_graph(['control[0]', 'control[1]', 'control[2]'],
                         colors3, ['Roll', 'Pitch', 'Yaw'])
     if not data_plot.had_error:
+        data_plot._p.y_range = Range1d(0, 0.01)
```

Comment:
Can you set `y_range=Range1d(0, 0.01)` in the constructor?

Reply:
Done.